### ISSUE TITLE
Iceberg does not work with latest stable Metacello

### ISSUE THREAD

**Issue by jecisc** [pharo-vcs/iceberg#260]
I wanted to use Iceberg in an image with the latest stable Metacello loaded (not the one of pharo, the one of metacello-work) but when I try to load a project I get: `MessageNotUnderstood: IceMetacelloRepositoryAdapter>>flushForScriptGet`

**Comment by estebanlm** [comment]
heh... and why this is a problem with iceberg and not metacello ;)

**Comment by dalehenrich** [comment]
I think that if you are modifying Metacello and not providing a pull request, _you_ are doing it wrong. I cannot read your mind, so I really need a pull request to kick things off.

I had an email conversation with @npasserini where we discussed this subject (not necessarily this specific bug) and I offered to make changes to Metacello when and if you guys found yourselves needing to change Metacello, so perhaps this is the time.

I am in the midst of some time-sensitive work right now (lots of work and a dead-line), so now is not a good time for me to have to do too much Metacello development, but I would definitely have time to review a pull request.
